gcloud/pubsub/subscription.py

```diff
@@ -259,18 +259,16 @@ def acknowledge(self, ack_ids, client=None):
                        ``client`` stored on the current subscription's topic.
         """
         client = self._require_client(client)
-        data = {'ackIds': ack_ids}
-        client.connection.api_request(
-            method='POST', path='%s:acknowledge' % (self.path,), data=data)
+        client.connection.subscription_acknowledge(self.full_name, ack_ids)
 
-    def modify_ack_deadline(self, ack_id, ack_deadline, client=None):
+    def modify_ack_deadline(self, ack_ids, ack_deadline, client=None):
```

Comment:
> Why is it needed?

The actual, underlying API request takes a list.

> Do the system tests need updating?

Nope, we don't actually exercise that one in system tests.

> Shouldn't it be renamed to modify_ack_deadlines?

Interesting point.  Grammatically, we are supplying *a* deadline for a *collection* of ack IDs.  I think in this case sticking to the unCamelCased name of the API method is more sensible.